families/battleship/tests/test_tp_battleship.py

```diff
@@ -15,21 +15,30 @@
 
 import logging
 import json
-
 from sawtooth_sdk.protobuf.validator_pb2 import Message
-from sawtooth_processor_test.transaction_processor_test_case \
```

Comment:
Since this is significant changes, you should add a bitwise copyright underneath the intel on